doc/src/compute_pressure.txt

```diff
@@ -16,12 +16,13 @@ ID, group-ID are documented in "compute"_compute.html command
 pressure = style name of this compute command
 temp-ID = ID of compute that calculates temperature, can be NULL if not needed
 zero or more keywords may be appended
-keyword = {ke} or {pair} or {bond} or {angle} or {dihedral} or {improper} or {kspace} or {fix} or {virial} :ul
+keyword = {ke} or {pair} or {bond} or {angle} or {dihedral} or {improper} or {kspace} or {fix} or {virial} or {hybridpair} :ul
 
 [Examples:]
 
 compute 1 all pressure thermo_temp
-compute 1 all pressure NULL pair bond :pre
+compute 1 all pressure NULL pair bond
+compute 1 all pressure NULL hybridpair lj/cut :pre
```

Comment:
more LAMMPS-like name for this keyword would be pair/hybrid, i.e. general/specific